### DISCUSSION TITLE
vue support ?

### DISCUSSION THREAD

**Discussion by johnqiuwan** [electric-sql/electric#1683] [Feature requests]
does any doc has the integration guide using vue project

**Comment by KyleAMathews** [comment]
We haven't worked on that yet. I know @tylerflint has expressed interest in developing one. If you look at the React integration though — it's fairly minimal as the react hook just subscribes to shape changes and then updates its snapshot of the data. https://github.com/electric-sql/electric/blob/main/packages/react-hooks/src/react-hooks.tsx

We'd love to see an integration PRed in!

**Reply by johnqiuwan** [reply]
Thank you for the quick reply!
